topics/instructors/tutorials/galaxy-admin-training/tutorial.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: tutorial_hands_on
+
+title: Galaxy Admin Training
+time_estimation: 60m
+questions:
+  - How do I prepare for a Galaxy Admin Training (GAT)
+  - What do I need to setup
+  - What should I know during the training?
+objectives:
+  - Interact with the UseGalaxy.eu admins to arrange for infrastructure
+  - Run a great training!
+key_points:
+  - Infrastructure is available for running GATs for free from UseGalaxy.eu
+  - This can be very convenient and easy to use
+  - EU provides appropriate DNS entries so you can run trainings with ITs.
+contributors:
+  - hexylena
+---
+
+# Introduction
+{:.no_toc}
+
+![GAT logo is the GTN logo over a space background and text reading galaxy admin traiing.](../../images/gat.png)
+
+Setting up and running a Galaxy Admin Training is not a very complicated process thanks to a significant amount of work that has been put into making it easy and quick to setup and run.
+
+This tutorial has multiple audiences who are all adressed within this one tutorial. We encourage everyone involved in hosting the GAT event to read this, so you are aware of all of the moving parts which are required for such an event.
+
+> ### Agenda
+>
+> In this tutorial, we will see:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Planning Your Training
+
+First consider the requirements for your training:
+
+- How many days will the training be?
+- What topics do you want to cover?
+	- Do you have enough helpers?
+	- Is the training online or in person?
+- Do you have your own infrastructure, or do you need to request infrastructure?
+
+We recommend [checking out an example schedule from the GAT repository](https://github.com/galaxyproject/admin-training/tree/2020-barcelona) and modifying it based on your needs.
+
+## Requesting Infrastructure
+
+If you do not have a cloud available that can, at minimum:
+
+- setup VMs which are accessible from where your students are
+- expose ports 20, 80, 443 (optional), 8080 (optional but nice)
+
+Then consider [contacting UseGalaxy.eu](mailto:galaxy@informatik.uni-freiburg.de?subject=Hosting%20a%20GAT) and let us know you'd like to host a GAT event. Tell us the number of students and dates of the event.
+
+# Provisioning the Infrastructure
+
+Skip to the section on setting up below that is appropriate for you
+
+## [EU admins] Setting up VMs
+
+> ### {% icon hands_on %} EU: Setting up VMs
+>
+> 1. Open a PR editing the [count variable](https://github.com/usegalaxy-eu/infrastructure/blob/master/training-vms.tf#L1) and increasing it to the desired number of VMs.
+>
+> 2. Merge
+>
+> 3. Once deployed, clone/pull the repository locally and run `./bin/process-training-output.sh`
+>
+>    ```
+>    $ ./bin/process-training-output.sh
+>    ubuntu  gat-0.training.galaxyproject.eu ...
+>    ubuntu  gat-1.training.galaxyproject.eu ...
+>    ```
+>
+> 4. Place this in the [GAT Machines](http://gxy.io/gatmachines) spreadsheet in the correct tab.
+>
+{: .hands_on}
+
+## [Non-EU admins] Setting up the VMs
```

Comment:
Maybe add a link to https://github.com/usegalaxy-eu/infrastructure/blob/master/training-vms.tf as an example for people using terraform?